testing/api/test_centerline.py

```diff
@@ -116,9 +116,9 @@ def dummy_centerline(size_arr=(9, 9, 9), pixdim=(1, 1, 1), subsampling=1, dilate
      {}),
     (dummy_centerline(size_arr=(30, 20, 50), subsampling=1, outlier=[20]),
      {'median': 0, 'rmse': 0.8, 'laplacian': 70, 'norm': 14},
-     {'exclude_nurbs': True}),
```

Comment:
It looks like the `exclude_nurbs` check that I thought was unnecessary still seems to be necessary. Even though it passed for me locally on Ubuntu 24.04, it still seems to be failing for CentOS, Ubuntu 22.04, etc. so I'll re-add it.

Reply:
Rebased!